docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md

```diff
@@ -4,6 +4,11 @@
 CREATE {TAG | EDGE} INDEX [IF NOT EXISTS] <index_name> ON {<tag_name> | <edge_name>} (prop_name_list)
 ```
 
+`prop_name_list`:
+
+- To index a variable string type property, you must use the `col_name(length)` syntax to specify an index prefix length
+- To index a fixed length string type property, you must use the `col_name` syntax because the string length can be read from the property
```

Comment:
col_name --> prop_name? 

an index prefix length --> the prefix length for an index?

To index a variable string type property --> To index a property of a variable-length string?

To index a fixed length string type property --> To index a property of a fixed-length string? 

you must use the `col_name` syntax because the string length can be read from the property --> only `prop_name` is necessary because the string length can be read from the property?